packages/expo/src/MailComposer/ExponentMailComposer.web.ts

```diff
@@ -0,0 +1,33 @@
+import array from 'cast-array';
+import filter from 'object-filter';
```

Comment:
Use https://www.npmjs.com/package/lodash.filter since others working on the Expo SDK are more likely to use it, it likely has more maintained type definitions, and it's maintained and used by lots of people who care about it working well and not being too large.